ecs-init/docker/docker.go

```diff
@@ -220,6 +220,11 @@ func (c *Client) getContainerConfig() *godocker.Config {
 		"ECS_ENABLE_TASK_IAM_ROLE_NETWORK_HOST": "true",
 	}
 
+	// for al, al2 add host ssl cert directory envvar if available
+	if config.HostCertsDirPath() != "" {
```

Comment:
nit: save the stat calls, `if certdir := config.HostCertsDirPath(); certdir != "" { }`

Reply:
oh right, that changed now. good catch thanks.